k8s-readonly.tf

```diff
@@ -29,6 +29,9 @@ resource "kubernetes_deployment" "apiary_hms_readonly" {
           name = "${local.hms_alias}-readonly"
         }
         annotations = {
+          "ad.datadoghq.com/hms-readonly.check_names": "[\"prometheus\"]"
+          "ad.datadoghq.com/hms-readonly.init_configs": "[{}]"
```

Comment:
I was thinking to have a variable(s) containing all metric names. If you have individual variables for each metric, then you have the same problem. You add a new one and you need to remember to modify both RO and RW.